front/src/test-data/geojson.ts

```diff
@@ -0,0 +1,52 @@
+import { random } from 'lodash';
+import type { LineString, Feature, Point, Position } from 'geojson';
+import { lineString as turfLineString } from '@turf/helpers';
+
+/**
+ * Creates a random position from a range
+ * @param valueRange the range [min, max] where the values belong
+ */
+export function position(valueRange: [number, number]): Position;
+/**
+ * Creates a json position from x and y values
+ * @param x x coordinate
+ * @param y y coordinate
+ */
+export function position(x: number, y: number): Position;
+export function position(arg1: number | [number, number], arg2?: number): Position {
+  if (Array.isArray(arg1)) {
+    return [random(arg1[0], arg1[1]), random(arg1[0], arg1[1])];
+  }
+  if (!arg2) {
+    throw new Error('y coordinate is not optional');
+  }
+  return [arg1, arg2];
+}
```

Comment:
This is not used for the moment, I don't really want to add unused code

```suggestion
```

Reply:
used in `lineString`

Reply:
OK then you can remove the export tag

Reply:
right